From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: aswin-mcw <aswin.kalies@multicorewareinc.com>
Date: Wed, 9 Sep 2026 13:13:38 +0530
Subject: [PATCH 003/109] newlib: libc: define _ctype for AArch64 Cygwin

(cherry picked from commit 09588b4cc3cafdc937c3ad6e892e5fbc0eb33bf1)
---
 newlib/libc/ctype/ctype_.c | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/newlib/libc/ctype/ctype_.c b/newlib/libc/ctype/ctype_.c
index 32ce4f318..869945fd5 100644
--- a/newlib/libc/ctype/ctype_.c
+++ b/newlib/libc/ctype/ctype_.c
@@ -95,21 +95,12 @@ char _ctype_b[128 + 256] = {
 /* For backward compatibility */
 char __EXPORT *__ctype_ptr__ = DEFAULT_CTYPE_PTR;
 
-#    ifdef __x86_64__
 __asm__ ("					\n\
         .data					\n\
 	.globl  _ctype_				\n\
 	.set    _ctype_,_ctype_b+127		\n\
 	.text                                   \n\
 ");
-#    else
-__asm__ ("					\n\
-        .data					\n\
-	.globl  __ctype_			\n\
-	.set    __ctype_,__ctype_b+127		\n\
-	.text                                   \n\
-");
-#    endif
 #  else /* !__CYGWIN__ */
 
 const char _ctype_[1 + 256] = {
